proto/one/operations/model/model_execution.proto

```diff
@@ -11,14 +11,18 @@ message ModelExecution {
     string version = 2;
     // Twin reference id associated with location or operation being modeled
     string twinRefId = 3;
-    //Model id
+    // Model id
     string externalModelId = 4;
-    //Length of forecast in hours
-    double forecastDurationHours = 5;
-    //Current inputs to the model
-    repeated ExecutionParameter inputs = 6;
-    //Current outputs for the model
-    repeated ExecutionParameter outputs = 7;
```

Comment:
Why move the inputs and outputs to a new index, rather than append the new fields?

Reply:
Because my brain thinks that physical location of code makes a difference.  Since this is not currently used I took the opportunity to organize the fields into what seems "correct"